sinks/gcm/driver.go

```diff
@@ -42,6 +42,15 @@ type gcmSink struct {
 	// TODO(vmarmol): Also store labels?
 	// Map of metrics we currently export.
 	exportedMetrics map[string]sink_api.MetricDescriptor
+
+	// TODO(vmarmol): Garbage collect these.
+	// Time we last exported a particular timeseries.
+	lastExportedTime map[lastExportedKey]time.Time
```

Comment:
This might be worth doing in the [central decoder](https://github.com/GoogleCloudPlatform/heapster/blob/master/sinks/api/decoder.go#L102)